### PR TITLE
Begin removing monolith query code

### PR DESCRIPTION
**Pull request by emord**
##### SUMMARY
This begins removing the monolith code. I haven't gotten around to actually `rm`ing those files or change the SQL output tests, but I think that's the main thing that's left.The agg should be using the distributed code only after this PR

### CODE REVIEW

File: custom/icds_reports/utils/aggregation_helpers/distributed/agg_ls_data.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
 from corehq.apps.userreports.util import get_table_name
 from custom.icds_reports.const import AGG_LS_VHND_TABLE, AGG_LS_AWC_VISIT_TABLE, AGG_LS_BENEFICIARY_TABLE
 from custom.icds_reports.utils.aggregation_helpers import transform_day_to_month, month_formatter
-from custom.icds_reports.utils.aggregation_helpers.monolith.base import BaseICDSAggregationHelper
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import BaseICDSAggregationDistributedHelper
 
 
-class AggLsHelper(BaseICDSAggregationHelper):
+class AggLsHelper(BaseICDSAggregationDistributedHelper):
```
Comment (czue) [inline custom/icds_reports/utils/aggregation_helpers/distributed/agg_ls_data.py]: why was this not distributed already?

File: custom/icds_reports/utils/aggregation_helpers/distributed/awc_infrastructure.py [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 from custom.icds_reports.const import AGG_INFRASTRUCTURE_TABLE
 from custom.icds_reports.utils.aggregation_helpers import month_formatter
-from custom.icds_reports.utils.aggregation_helpers.monolith.base import BaseICDSAggregationHelper
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import BaseICDSAggregationDistributedHelper
 
 
-class AwcInfrastructureAggregationHelper(BaseICDSAggregationHelper):
+class AwcInfrastructureAggregationHelper(BaseICDSAggregationDistributedHelper):
```
Comment (czue) [inline custom/icds_reports/utils/aggregation_helpers/distributed/awc_infrastructure.py]: same question

**Review by czue — APPROVED** [review]
yes! i think this will solve 90% of my issues. just a couple questions for my own education.

**Review by Rohit25negi — APPROVED** [review]
Looks good. Left one point. But I think the answer is true. Feel free to merge if so.

File: custom/icds_reports/utils/aggregation_helpers/distributed/agg_ls_data.py (hunk 1 above)
Comment (emord) [reply custom/icds_reports/utils/aggregation_helpers/distributed/agg_ls_data.py]: The same queries were run on both monolith and distributed implementations. The change/move I made was to make it easier to `rm -rf .../monolith/` later

See: https://github.com/dimagi/commcare-hq/pull/25604/files#diff-7fb91312bce0449a4bd7cc9e51062aedL179
https://github.com/dimagi/commcare-hq/blob/a3387e4cb89c41fdc7305e92a8fc97e9a8bc8228/custom/icds_reports/utils/aggregation_helpers/helpers.py#L87-L92

**Pull request merged**